frontend/src/app/assess/[projectId]/fairness-bias/page.tsx

```
@@ -486,27 +482,27 @@ export default function FairnessBiasTest() {
-                  <h2 className="text-xl font-semibold text-gray-900 dark:text-white leading-relaxed">
+                  <h2 className="text-xl font-semibold text-foreground leading-relaxed">
                     {currentPrompt.text}
                   </h2>
                 </div>
 
                 <div className="mb-6">
-                  <label className="block text-sm font-medium text-gray-700 dark:text-gray-300 mb-2">
+                  <label className="block text-sm font-medium text-foreground mb-2">
                     Your Response
                   </label>
                   <textarea
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)